src/MoBi.Assets/AppConstants.cs

```diff
@@ -1878,6 +1878,20 @@ public static string AlsoImportIndividualsAndExpressions(int numberOfIndividuals
 
             return $"Also import {individual} and {expression}?";
          }
+
+         public static string CouldNotAddExpressionsDuplicatingMolecule(IReadOnlyList<string> allNames)
+         {
+            if(allNames.Count == 1)
+               return $"The expression could not be added because an expression is already selected for the protein{Environment.NewLine}{namesList(allNames)}";
```

Comment:
I think we should use the molecule name, Right now it's the expression profile name? Or the text should change maybe? Not sure

Reply:
just put something like

``` 
Expression profiles cannot be added for
 - CYP3A4
 - whatever else

because an existing expression profile is already selected for those proteins
```

Reply:
yeah makes sense